python/ray/_raylet.pyx

```diff
@@ -3336,6 +3336,13 @@ cdef class CoreWorker:
 
         return RayObjectsToDataMetadataPairs(results)
 
+    def get_release(self, object_refs):
```

Comment:
why don't we call get_read_release? The name kind of sounds confusing with "free" API to me